k8s/test/e2e/repartition-demo/Dockerfile.dask-worker-with-vineyard

```diff
@@ -1,3 +1,3 @@
 FROM ghcr.io/dask/dask:2022.8.1-py3.10
 
-RUN pip3 install --no-cache-dir vineyard vineyard-dask
+RUN pip3 install --no-cache-dir vineyard==0.11.7 vineyard-dask==0.11.7
```

Comment:
No need to fixes the version.

Some codecy warnings could be ignored. Otherwise we would need to update this file again and again, that's boring.

Reply:
Ok.